Add warning about eigenvalues of Ptilde

cc7768 opened the pull request:
Kills asset pricing if the primitives do not satisfy that the modulus of the eigenvalues of Ptilde are less than 1.

oyamad commented on markov_asset/asset_pricing.py:
```diff
@@ -82,6 +82,14 @@ def __init__(self, beta, P, s, gamma):
+        # Create P_tilde
+        Ptilde = self.P_tilde
+        ev = eigvals(beta*Ptilde)
+        lt1 = np.all(np.absolute(ev) < 1.0)
```
Minor comment:
`Ptilde` is a nonnegative matrix, so its dominant eigenvalue is real, and hence it is sufficient to compare the real part:

```python
lt1 = np.all(ev.real < 1.0)
```

In fact, it seems even without `real` the real part is compared:

```python
lt1 = np.all(ev < 1.0)
```

oyamad commented on markov_asset/asset_pricing.py:
```diff
@@ -82,6 +82,14 @@ def __init__(self, beta, P, s, gamma):
 
+        # Create P_tilde
+        Ptilde = self.P_tilde
+        ev = eigvals(beta*Ptilde)
```
Very minor comment:
Here multiplication occurs nxn times, whereas if you do `ev = eigvals(Ptilde) * beta` it's n times.

oyamad commented on markov_asset/asset_pricing.py:
```diff
@@ -82,6 +82,14 @@ def __init__(self, beta, P, s, gamma):
+        ev = eigvals(beta*Ptilde)
+        lt1 = np.all(np.absolute(ev) < 1.0)
+        if not lt1:
+            msg = "All eigenvalues of Ptilde must be less than 1"
```
"less than 1/beta"?

mmcky commented:
@cc7768 Does this PR need to be updated based on the feedback above?

cc7768 commented:
Yes. I had forgotten about this -- Will fix and update today.

Thanks @oyamad and @mmcky

spencerlyon2 commented:
Ping! Let's get to 0 PRs!

cc7768 commented:
I addressed each of @oyamad comments -- Thank you for the feedback.

Also rebased wth master and made sure that the notebook ran as well (using my version not the master branch version -- I left the notebook as is though, so that when it is merged everything should just work).
